migrations/20160519162112-make-group.slug-unique-migration.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface) {
+    return queryInterface.addIndex('Groups', ['slug'], {
```

Comment:
Are you sure you need an entirely separate index to enfore slug unicity? Can't it be an index on the combination of ID and slug columns or similar? Not super familiar with SQL details, but in general the least indices, the better the performance.